app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    """ That will move a file from one location to another """
+    # Ex: mv file.txt first_dir/second_dir/third_dir/file2.txt
+
+    (cmd,
+     file_in,
+     file_out) = (
+        command.split(" ")[0],
+        command.split(" ")[1],
+        command.split(" ")[2])
+
+    if cmd != "mv":
+        raise ValueError("Command mv not found")
+
+    if "/" in file_out:
+        dir_path = ""
+
+        for directory in os.path.dirname(file_out).split("/"):
+            dir_path = os.path.join(dir_path, directory)
+
+            if not os.path.exists(dir_path):
+                os.makedirs(dir_path)
```

Comment:
You can simplify this code with using os.makedirs(dir_path, exist_ok=True)
```suggestion
            os.makedirs(dir_path, exist_ok=True)
```